### PR TITLE
feat(ai-summary): add topic summary to discussion thread

### PR DESCRIPTION
**Pull request by nickoferrall**
Fix https://github.com/ParabolInc/parabol/issues/7472

An edge case: if you create the groups in the Group phase and race through to the Discuss Phase, we may not have received the response from OpenAI with the topic summary. It's unlikely that this will happen as most teams in real meetings will spend some time in the Vote phase, but if it does happen, they'll just see an empty discussion thread. 

Demo: https://www.loom.com/share/1512b7ca727548138e492bb864af7d37

### To test

- [ ] Make sure that you've added the `OPEN_AI_API_KEY` and `OPEN_AI_ORG_ID` env vars
- [ ] Add the aiSummary feature flag to your user
- [ ] Create a retro that includes groups with several reflections as well as groups with just one reflection
- [ ] See that the discussion thread includes a comment from Parabol AI that summarises the topic for groups with two or more reflections
- [ ] If a group only has one reflection, there is no topic summary

### CODE REVIEW

File: packages/client/modules/email/components/SummaryEmail/MeetingSummaryEmail/RetroTopic.tsx [hunk 1]
```diff
@@ -133,23 +133,23 @@ const RetroTopic = (props: Props) => {
       {(topicSummary || discussionSummary) && (
         <tr>
           <td align='left' style={{lineHeight: '22px', fontSize: 14}}>
-            {topicSummary && (
+            {team?.tier === 'personal' && (
               <tr>
-                <td style={topicTitleStyle}>{'Topic Summary:'}</td>
+                <td
+                  style={explainationStyle}
```
Comment (nickoferrall) [inline packages/client/modules/email/components/SummaryEmail/MeetingSummaryEmail/RetroTopic.tsx:139]: I've moved this around as I think the explanation sentence should come before the topic & discussion summary titles

**Review by igorlesnenko — APPROVED** [review]
Looks good!

File: packages/server/postgres/migrations/1670438374119_addAIUser.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import {Client} from 'pg'
+import getPgConfig from '../getPgConfig'
+
+export async function up() {
+  const client = new Client(getPgConfig())
+  await client.connect()
+  const picture = 'https://action-files.parabol.co/static/favicon.ico'
```
[the remark this answers is not in the document]
Comment (Dschoordsch) [reply packages/server/postgres/migrations/1670438374119_addAIUser.ts:7]: +1 If you find out how to gain access, could you please also update the picture of `aGhostUser`?
https://github.com/ParabolInc/parabol/blob/db6909c09d2f0ea3518481bc9612fbc686ae6c56/packages/server/postgres/migrations/1631553388387_add-template-teams-users.ts#L56

**Review by Dschoordsch — CHANGES_REQUESTED** [review]
Not as part of this PR, but I think we need to tune in the AI query. At the moment it seems it's oversimplifying too much
Screenshot from 2022-12-14 10-58-45

**Comment by nickoferrall** [comment]
Completely agree about fine-tuning it. I've created an issue: https://github.com/ParabolInc/parabol/issues/7569

**Review by Dschoordsch — APPROVED** [review]

**Comment by Dschoordsch** [comment]
I think even before we go full fledged custom model, we should play around with the summary request and maybe exchange "to a second grade student" with something else. Or just go "TL;DR"

**Comment by nickoferrall** [comment]
> I think even before we go full fledged custom model, we should play around with the summary request and maybe exchange "to a second grade student" with something else. Or just go "TL;DR"

Yeah, I think it would be a good thing to A/B test. We could implement the up/down arrows (mentioned in https://github.com/ParabolInc/parabol/issues/7569) and see which prompts & text inputs get the best results

**Pull request merged**